### PR TITLE
PGB-1273 Add analyse db documentation for roles/permissions for basis…

### PR DESCRIPTION
**Pull request by JKoetsier**
…informatie users

### CODE REVIEW

File: docs/analyse_database/README.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+2. Grant agm_basis or agm_plus (or both) to the new user:
+    ```
+    GRANT agm_basis/agm_plus to username;
+    ```
+
+That's it!
+
+## Initial setup
+Run both both default_roles.sql and maatwerkbasisinformatie_permissions.sql on the new analyse database.
```
Comment (jasper-jager) [inline docs/analyse_database/README.md:61]: both both :)

File: docs/analyse_database/README.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+## Design
+### Schemas and default roles
+All registrations in GOB have their own schemas in the analyse database. There is a bag schema, brk schema,
+gebieden, etc. For each schema there is a corresponding _read role (bag_read, brk_read, gebieden_read,
+etc). This read role has, as the name suggests, read permissions on the schema. There are no write roles
+for these schemas.
+
+There is one extra schema called 'maatwerkbasisinformatie', where (some) basisinformatie employees have
+write permissions. This is their playground where they can place the results of their analyses. Two roles
```
Comment (OnnoLogchies) [inline docs/analyse_database/README.md:14]: Another purpose of this schema is to create views of frequently used queries or joins which can be used by everyone

File: docs/analyse_database/README.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+are granted either only the agm_basis role or both the agm_basis AND agm_plus roles.
+
+- The agm_basis role provides read access to all registrations except BRK and read access to the
+maatwerkbasisinformatie schema.
+- The agm_plus role provides read access to BRK and write access to the maatwerkbasisinformatie schema.
+
+## SQL scripts in this directory
+### default_roles.sql
+This file contains the SQL to create the default roles, including agm_basis and agm_plus. The SQL in this
+file should only have to be run when creating a new analyse database from scratch. Only when a new
```
Comment (OnnoLogchies) [inline docs/analyse_database/README.md:30]: this script should be executed by the user that creates the tables, gob_analyse in our case. If another user executes this script the read permissions will not be set automatically on table creation

File: docs/analyse_database/README.md [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+    ```
+    CREATE USER username PASSWORD '********' 
+    -- (Make sure to replace username with the correct username and ******* with an actual password.)
+    ```    
+
+2. Grant agm_basis or agm_plus (or both) to the new user:
+    ```
+    GRANT agm_basis/agm_plus to username;
+    ```
+
```
Comment (OnnoLogchies) [inline docs/analyse_database/README.md:57]: 3. Preferably make sure that the user changes his or her password after the first login. Unfortunately this is not an option which can be forced in Postgres

File: docs/analyse_database/README.md (hunk 2 above)
Comment (JKoetsier) [reply docs/analyse_database/README.md]: Mentioning the views now too.

File: docs/analyse_database/README.md (hunk 3 above)
Comment (JKoetsier) [reply docs/analyse_database/README.md:31]: Good one, added :)

File: docs/analyse_database/README.md (hunk 4 above)
Comment (JKoetsier) [reply docs/analyse_database/README.md:60]: Good one too, added point 3 with an example.

**Pull request merged**